assets/css/evaluation.less

```diff
@@ -0,0 +1,71 @@
+body {
```

Comment:
Please fix the structure/skin separation everywhere. Currently we've done in a very limited way